crates/testing/tests/tests_1/consensus_task.rs

```diff
@@ -65,7 +64,7 @@ async fn test_consensus_task() {
             quorum_proposal_validated(),
             exact(QuorumVoteSend(votes[0].clone())),
         ],
-        asserts: vec![is_at_view_number(1)],
+        asserts: vec![],
```

Comment:
Were these asserts removed accidentally, or were they conceptually wrong?

Reply:
Yeah they honestly just didn't make sense to have since we're basically replicating what a view change is. I made this edit on day 2 here, so I had *fundamentally* no idea what was going on.